README.md

```diff
@@ -305,15 +309,15 @@ this input.
 
 Let us confirm this is the issue.
 
+[//]: # (todo why is the runtime API used when there is a dedicated task class?)
```

Comment:
@akshayabd just as the `todo` asks.

Reply:
Mainly to give them some more practice using the Runtime API. Sometimes build caching tweaks will have to be made using the Runtime API when they don't have access to the task class, so it's important users get familiar with it.

Reply:
I agree with that general sentiment, but in this specific case I don't think it should be used. Understanding how to add inputs / outputs to proper task classes, having familiarity with the right annotations, etc. is equally as important.

In this case since there is already a task class, I say use the task class. I think it's confusing to mix and match using the runtime API and task classes. It may also give the wrong impression about how to deal these cache misses.

Reply:
We discussed this and agreed not to use the runtime API here. 

We are considering creating a plugin with intentional build cache misses to be used here and possibly in other trainings.